data/Psi4/basicPsi4-1.2.1/water_mp2.out

```diff
@@ -1,9 +1,9 @@
 
     -----------------------------------------------------------------------
           Psi4: An Open-Source Ab Initio Electronic Structure Package
-                               Psi4 1.2.1 release
+                               Psi4 1.3.2 release
```

Comment:
This should not be in the `basicPsi4-1.2.1` folder correct? Either modify the psi4 1.3.1 tests or it can be added as a regression test

Reply:
No -- I didn't notice this. I'll modify the 1.3.1 test instead and add it back. Thanks for letting me know about this!

Reply:
I've replaced this with 1.2.1 output file. Thanks!